src/main/java/com/javahelps/restservice/CodeReview.java

```diff
@@ -1 +1,42 @@
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.Locale;
 
+public class CodeReview {
+
+    Integer x;
```

Comment:
variable should have an obvious name represent what data will be store in.
Could you please clarify it?